session/session.go

```diff
@@ -1143,6 +1143,9 @@ func (s *session) SetGlobalSysVar(name, value string) (err error) {
 	if value, err = sv.Validate(s.sessionVars, value, variable.ScopeGlobal); err != nil {
 		return err
 	}
+	if sv.Name == "tidb_evolve_plan_baselines" && strings.EqualFold("ON", value) && !config.CheckTableBeforeDrop {
```

Comment:
Better to use the variable `TiDBEvolvePlanBaselines` instead of a literal string `tidb_evolve_plan_baselines`.